src/Util/CloneTrait.php

```diff
@@ -14,7 +14,7 @@
 namespace ApiPlatform\Core\Util;
 
 /**
- * Clones given data if cloneable.
+ * Clones given data if clonable.
```

Comment:
"Cloneable" is correct: https://www.php.net/manual/en/reflectionclass.iscloneable.php

Please revert.

Reply:
Thanks, reverted!